README.md

```diff
@@ -7,6 +7,22 @@ A Kubernetes Operator to automate deployment of
 [Cryostat](https://github.com/cryostatio/cryostat) and provide an
 API to manage [JDK Flight Recordings](https://openjdk.java.net/jeps/328).
 
+## SEE ALSO
+
+* [cryostat-core](https://github.com/cryostatio/cryostat-core) for
+the core library providing a convenience wrapper and headless stubs for use of
+JFR using JDK Mission Control internals.
+
+* [cryostat](https://github.com/cryostatio/cryostat)
+for an OpenShift Operator facilitating easy setup of Cryostat in your OpenShift
+cluster as well as exposing the Cryostat API as Kubernetes Custom Resources.
+
+* [cryostat-web](https://github.com/cryostatio/cryostat-web) for the JFR management service
```

Comment:
Er, should this be the text for `cryostat`? And for `-web` it might say something about the graphical frontend or browser client for Cryostat?

Reply:
Ugh, my bad.

Reply:
Adjusted in the latest commit